api/python/cellxgene_census/tests/experimental/ml/test_pytorch.py

```diff
@@ -12,19 +12,15 @@
 from somacore import AxisQuery
 from tiledbsoma import Experiment, _factory
 from tiledbsoma._collection import CollectionBase
-from torch.utils.data._utils.worker import WorkerInfo
 
 # conditionally import torch, as it will not be available in all test environments
-try:
-    from torch import Tensor, float32
+from torch import Tensor, float32
```

Comment:
TODO: remove this once we figure out the issue.

Reply:
Reset the commit hash to remove it